bin/resources/GameIndex.yaml

```diff
@@ -6388,7 +6388,7 @@ SCPS-55031:
   region: "NTSC-J"
 SCPS-55032:
   name: "Fantavision - For You and Me"
-  region: "NTSC-J"
+  region: "NTSC-Ch-E-J"
```

Comment:
Is this region valid?

Reply:
As far as I'm aware yes

Reply:
That's not valid in Qt, it might just pick China, if you're lucky, at worse it won't know wtf you're talking about.

Reply:
It comes up as China and there are already other entry's under it

Reply:
You're lucky then :)

Reply:
It was fine as it was.